packages/twenty-server/src/core/messaging/dtos/timeline-thread.dto.ts

```diff
@@ -12,6 +13,10 @@ export class TimelineThread {
   @Field()
   read: boolean;
 
+  @Field()
+  @IsEnum(['metadata', 'subject', 'share_everything'])
```

Comment:
@bosiraphael actually there is nothing to validate here, I originally thought this was an input type. IsEnum won't help you there.
If you want a proper enum type for your graphql schema, you can declare your type like this
```
export enum TimelineThreadVisibility {
  Metadata = 'metadata',
  Subject = 'subject',
  ShareEverything = 'share_everything',
}
```
then register it like thi
```
registerEnumType(TimelineThreadVisibility, {
  name: 'TimelineThreadVisibility',
});
```